spec/index.bs

```diff
@@ -15884,7 +15882,12 @@ It guarantees that:
 
 ### Sample frequency shading ### {#sample-frequency-shading}
 
-<p class="note editorial"><span class=marker>Editorial note:</span> fill out the section
+For multisampled render attachments fragment shaders can be run once per-pixel or once per-sample. Fragment shaders will
+run once per-sample if the `sample_index` [=builtin=] is used by the shader. Otherwise fragment shaders are run once
+per-pixel with the result broadcast out to each of the samples included in the [=final sample mask=].
+
+Note: When using per-sample shading the color output for sample |N| is produced by the fragment shader execution
```

Comment:
nit: I don't think this needs to be non-normative now that it's in this section. (Also comma)
```suggestion
When using per-sample shading, the color output for sample |N| is produced by the fragment shader execution
```